hack/patch-s2i-task.sh

```diff
@@ -1,21 +0,0 @@
-#!/usr/bin/env bash
-
-# This script patches the s2i Tekton task, so it recognizes registry.default.svc.cluster.local:5000 as insecure.
-
-echo "Patching s2i Tekton task to use 'registry.default.svc.cluster.local:5000' as an insecure registry."
-
-patch pkg/pipelines/resources/tekton/task/func-s2i/0.1/func-s2i.yaml <<EOF
```

Comment:
Note: this patch was not working anymore. We are pulling tekton tasks directly from PR branch via `https://raw.githubusercontent.com/` -- local changes have no effect.